### ISSUE TITLE
evdev device not accessible from unprivileged user?

### ISSUE THREAD

**Issue by Apteryks** [DanielOgorchock/joycond#148]
Hello,

I was testing the udev rules provided by this package with a Nintendo Switch Pro controller, and my understanding of it was that it should make it available for use as my normal (unprivileged) user, given this line: https://github.com/DanielOgorchock/joycond/blob/9d1f5098b716681d087cca695ad714218a18d4e8/udev/72-joycond.rules#L5C1-L5C90

```
ATTRS{id/vendor}=="057e", ATTRS{id/product}=="2009", ATTRS{name}=="*IMU*", TAG+="uaccess"
```
Connecting the device shows the following messages:
```
Sep 17 15:15:26 localhost vmunix: [ 1128.244218] usb 1-5: new full-speed USB device number 7 using xhci_hcd
Sep 17 15:15:26 localhost vmunix: [ 1128.643098] usb 1-5: New USB device found, idVendor=057e, idProduct=2009, bcdDevice= 2.10
Sep 17 15:15:26 localhost vmunix: [ 1128.643106] usb 1-5: New USB device strings: Mfr=1, Product=2, SerialNumber=3
Sep 17 15:15:26 localhost vmunix: [ 1128.643107] usb 1-5: Product: Pro Controller
Sep 17 15:15:26 localhost vmunix: [ 1128.643108] usb 1-5: Manufacturer: Nintendo Co., Ltd.
Sep 17 15:15:26 localhost vmunix: [ 1128.643109] usb 1-5: SerialNumber: 000000000001
Sep 17 15:15:27 localhost vmunix: [ 1128.669209] nintendo 0003:057E:2009.000C: hidraw10: USB HID v81.11 Joystick [Nintendo Co., Ltd. Pro Controller] on usb-0000:08:00.0-5/input0
Sep 17 15:15:27 localhost vmunix: [ 1129.556156] nintendo 0003:057E:2009.000C: controller MAC = 90:45:28:17:80:23
Sep 17 15:15:27 localhost vmunix: [ 1129.595842] nintendo 0003:057E:2009.000C: using factory cal for left stick
Sep 17 15:15:27 localhost vmunix: [ 1129.635852] nintendo 0003:057E:2009.000C: using factory cal for right stick
Sep 17 15:15:28 localhost vmunix: [ 1129.748194] nintendo 0003:057E:2009.000C: using factory cal for IMU
Sep 17 15:15:28 localhost vmunix: [ 1129.900195] nintendo 0003:057E:2009.000C: assigned player 1 led pattern
Sep 17 15:15:28 localhost vmunix: [ 1129.972338] input: Nintendo Co., Ltd. Pro Controller as /devices/pci0000:00/0000:00:02.1/0000:02:00.0/0000:03:0c.0/0000:08:00.0/usb1/1-5/1-5:1.0/0003:057E:2009.000C/input/input19
Sep 17 15:15:28 localhost vmunix: [ 1129.972464] input: Nintendo Co., Ltd. Pro Controller (IMU) as /devices/pci0000:00/0000:00:02.1/0000:02:00.0/0000:03:0c.0/0000:08:00.0/usb1/1-5/1-5:1.0/0003:057E:2009.000C/input/input20
Sep 17 15:15:28 localhost mtp-probe: checking bus 1, device 7: "/sys/devices/pci0000:00/0000:00:02.1/0000:02:00.0/0000:03:0c.0/0000:08:00.0/usb1/1-5" 
Sep 17 15:15:28 localhost mtp-probe: bus: 1, device: 7 was not an MTP device 
Sep 17 15:15:28 localhost shepherd[1]: [joycond] Removing /devices/pci0000:00/0000:00:02.1/0000:02:00.0/0000:03:0c.0/0000:08:00.0/usb1/1-5/1-5:1.0/0003:057E:2009.000B/input/input17/event16 from unpaired list 
Sep 17 15:15:28 localhost shepherd[1]: [joycond] DEVNAME=event16 ACTION=add DEVPATH=/devices/pci0000:00/0000:00:02.1/0000:02:00.0/0000:03:0c.0/0000:08:00.0/usb1/1-5/1-5:1.0/0003:057E:2009.000C/input/input19/event16 
Sep 17 15:15:28 localhost shepherd[1]: [joycond] Creating new phys_ctlr for /dev/input/event16 
Sep 17 15:15:28 localhost shepherd[1]: [joycond] Found Pro Controller 
Sep 17 15:15:28 localhost shepherd[1]: [joycond] driver_name: Nintendo Co., Ltd. Pro Controller 
Sep 17 15:15:28 localhost shepherd[1]: [joycond] MAC: 90:45:28:17:80:23 
Sep 17 15:15:28 localhost mtp-probe: checking bus 1, device 7: "/sys/devices/pci0000:00/0000:00:02.1/0000:02:00.0/0000:03:0c.0/0000:08:00.0/usb1/1-5" 
Sep 17 15:15:28 localhost mtp-probe: bus: 1, device: 7 was not an MTP device 
Sep 17 15:15:28 localhost shepherd[1]: [joycond] adding epoll_subscriber: fd=5
```

The evdev devices created for the controller are:
```
$ ls -al /dev/input/by-id/usb-Nintendo_Co.__Ltd._Pro_Controller_000000000001-event-*
lrwxrwxrwx 1 root root 10 17 sept. 15:15 /dev/input/by-id/usb-Nintendo_Co.__Ltd._Pro_Controller_000000000001-event-if00 -> ../event17
lrwxrwxrwx 1 root root 10 17 sept. 15:15 /dev/input/by-id/usb-Nintendo_Co.__Ltd._Pro_Controller_000000000001-event-joystick -> ../event16
```

and their permissions are;

```
$ ls -al /dev/input/event{16,17}
crw------- 1 root input 13, 80 17 sept. 15:15 /dev/input/event16
crw-rw---- 1 root input 13, 81 17 sept. 15:15 /dev/input/event17
```

In other words, only users part of the `input` group will be able to use the controller. Thus, it seems the 
`TAG+="uaccess"` doesn't work as intended?

Thanks for this free (as in libre) software!
